view/frontend/layout/checkout_cart_index.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceContainer name="checkout.cart.methods">
+            <block name="express.pay" template="Bold_CheckoutPaymentBooster::express-pay.phtml" after="checkout.cart.methods.onepage.bottom">
+                <arguments>
+                    <argument name="express_pay_view_model" xsi:type="object">Bold\CheckoutPaymentBooster\ViewModel\ExpressPay</argument>
+                </arguments>
+            </block>
+        </referenceContainer>
+        <move element="checkout.cart.totals.container" destination="cart.summary" before="checkout.cart.methods.bottom"/>
```

Comment:
We should not be moving core elements. Let's make note of this so that we can circle back to it.

Reply:
Created ticket [CHK-6923](https://boldapps.atlassian.net/browse/CHK-6923) to capture this note so we can circle back later